GroupIS1024/Models/Player.cs

```diff
@@ -27,11 +28,12 @@ public class Player
 
         public string University { get; set; }
 
+        [DataType(DataType.Date)]
```

Comment:
I have added datatype data annotation. As time is not required for the date of the birth property.